test/e2e/testing-manifests/pod

```diff
@@ -8,6 +8,6 @@ metadata:
 spec:
   containers:
   - name: nginx
-    image: nginx
```

Comment:
same as below.

Reply:
https://github.com/kubernetes/kubernetes/blob/4e30fe40df258d30dac6d2cc01bd75f175445ac8/test/e2e_node/e2e_node_suite_test.go#L99-L100
-  In `test/e2e_node/testing-manifests`, another image need to be migrated `docker.io/nfvpe/sriov-device-plugin:v3.1`.
- In `test/e2e/testing-manifests/pod` the nginx needs to updated